synapse/lib/view.py

```diff
@@ -41,12 +41,12 @@ async def __anit__(self, core, link, user, view):
         await s_cell.CellApi.__anit__(self, core, link, user)
         self.view = view
         layriden = view.layers[0].iden
-        self.allowedits = user.allowed(('layer', 'write'), gateiden=layriden)
+        self.allowedits = user.allowed(('node'), gateiden=layriden)
```

Comment:
Should you be able to fork a view with a mirrored layer?  Seems like it would work, but an eventual merge might add complexity not worth supporting at this point?